packages/create-svelte/scripts/build-templates.js

```diff
@@ -8,11 +8,18 @@ import { mkdirp, rimraf } from '../utils.js';
 
 /** @param {string} content */
 function convert_typescript(content) {
-	const transformed = transform(content, {
-		transforms: ['typescript']
+	let { code } = transform(content, {
+		transforms: ['typescript'],
+		disableESTransforms: true
 	});
 
-	return prettier.format(transformed.code, {
+	// sucrase leaves invalid class fields intact
+	code = code.replace(/^\s*[a-z]+;$/gm, '');
+
+	// Prettier strips 'unnecessary' parens from .ts files, we need to hack them back in
+	code = code.replace(/(\/\*\* @type.+? \*\/) (.+?) \/\*\*\*\//g, '$1($2)');
+
+	return prettier.format(code, {
```

Comment:
I remember having the same problem in `prettier-plugin-svelte` (the parentheses thing) when using the `typescript` parser (https://github.com/sveltejs/prettier-plugin-svelte/issues/218), using the `babel-ts` parser fixed it. I wonder though why it doesn't work in this case, because the formatter uses the `babel` parser, which I would have expected to be a JS parser which should keep the parentheses.